index.html

```diff
@@ -39,19 +39,42 @@
         <h2>Controls</h2>
         <h3>Date Range</h3>
         <form name="dateSelection">
-          <div class="dateRadio">
+          <div class="radioBtn">
```

Comment:
Since these we assign an onclick for this, rather than have some kind of form submission, these could probably be buttons. An inline button group like https://v4-alpha.getbootstrap.com/components/button-group/ would look really neat